package.json

```diff
@@ -53,31 +55,26 @@
   "scripts": {
     "prepare": "husky",
     "start": "node ./scripts/start-dev-server.js",
-    "clean": "find . -name \"node_modules\" -type d -prune | xargs rm -rf",
+    "lint": "npm run lint:scripts && npm run lint:styles",
+    "lint:scripts": "eslint \"**/*.{js,jsx,ts,tsx,mjs}\"",
+    "lint:styles": "stylelint \"**/*.scss\"",
+    "lint:staged": "lint-staged --verbose",
+    "build": "nx run @progress/kendo-themes-html:build",
```

Comment:
You can define short package names by adding a `project.json` to the respective package folder, e.g.:

**packages/html/package.json**
```json
{
  "name": "html",
  "$schema": "../../node_modules/nx/schemas/project-schema.json",
  "sourceRoot": "packages/html/src",
  "projectType": "library",
  "tags": []
}
```